src/app/templates/activity/Activity.tsx

```diff
@@ -40,7 +40,7 @@ export const ActivityComponent: React.FC<Props> = ({ assetSlug }) => {
         )}
       >
         <div className="w-full flex justify-center mb-6">
-          <PartnersPromotion variant={PartnersPromotionVariant.Image} />
+          <PartnersPromotion id="promo-activity-item" variant={PartnersPromotionVariant.Image} />
```

Comment:
Now if user hides the ad on activity page it will also hide on all tokens pages (cause the same component is used there), is it ok?

Reply:
As far as I know, it doesn't work this way because timestamps of hiding the ad are selected by promotion id, which is different for each banner.

Reply:
I don't know

Reply:
See https://madfishgroup.slack.com/archives/C0535E9N3PE/p1702466203612429 thread